src/main/java/io/vertx/core/net/impl/ConnectionBase.java

```diff
@@ -491,7 +497,10 @@ public SocketAddress remoteAddress() {
   public SocketAddress localAddress() {
```

Comment:
Do we need todocument localAddress as well? Just like we did for remote address?

Reply:
yes, I forgot this

Reply:
yes

Reply:
Ok will do that now and remove the hardcoded version.Thanks